src/test/java/io/swagger/codegen/v3/generators/options/JavaOptionsProvider.java

```diff
@@ -74,6 +74,7 @@ public JavaOptionsProvider() {
                 .put("hideGenerationTimestamp", "true")
                 .put(CodegenConstants.ALLOW_UNICODE_IDENTIFIERS, ALLOW_UNICODE_IDENTIFIERS_VALUE)
                 .put(CodegenConstants.USE_OAS2, "true")
+                .put("notNullJacksonAnnotation", "true")
```

Comment:
what is this about? how is this used? the default for `notNullJacksonAnnotation` should be `false`